packages/@sfcc-core/core-graphql/src/index.ts

```diff
@@ -4,20 +4,34 @@
     SPDX-License-Identifier: BSD-3-Clause
     For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
 */
-import apolloServerExpress from 'apollo-server-express';
+import apolloServerExpress, {
+    ApolloServer as ApolloServerType,
+} from 'apollo-server-express';
```

Comment:
This is some weird stuff. We can clean it up if we end up migrating the storefront to TypeScript

Reply:
We maybe could breakdown this source code into another file(s) and use index.ts as a "barrel" for future enumerated exports. (this comment unrelated to apolloserver import)